### PR TITLE
Create instances out of event war. War Revamp 2021.

### PR DESCRIPTION
**Pull request by LlmDl**
#### Description: 

Details to follow...

____
#### New Nodes/Commands/ConfigOptions: 

____
#### Relevant Towny Issue ticket:

____
- [ ] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

File: src/com/palmergames/bukkit/towny/tasks/OnPlayerLogin.java [hunk 1]
```diff
@@ -129,7 +129,7 @@ public void run() {
 			}
 
 		if (TownyAPI.getInstance().isWarTime()) {
-			universe.getWarEvent().sendScores(player, 3);
+			universe.getWarEvent(player).getScoreManager().sendScores(player, 3);
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/tasks/OnPlayerLogin.java:132]: Potentially list all active wars.

File: src/com/palmergames/bukkit/towny/object/TownyUniverse.java [hunk 2]
```diff
@@ -167,7 +137,7 @@ public static void jailTeleport(final Player player, final Location location) {
 	 */
```
Comment (TheFlagCourier) [inline src/com/palmergames/bukkit/towny/object/TownyUniverse.java:137]: `{@link TownyWorld#addWarZone(Coord)}` is unresolvable.

File: src/com/palmergames/bukkit/towny/object/TownyUniverse.java [hunk 3]
```diff
@@ -176,6 +146,6 @@ public void addWarZone(WorldCoord worldCoord) {
...
```
Comment (TheFlagCourier) [inline src/com/palmergames/bukkit/towny/object/TownyUniverse.java:146]: `{@link TownyWorld#removeWarZone(Coord)}` is unresolvable.

File: src/com/palmergames/bukkit/towny/war/eventwar/WarBooks.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+public class WarBooks {
+
+	private static String newline = "\n";
+	private static final SimpleDateFormat warDateFormat = new SimpleDateFormat("MMM d yyyy '@' HH:mm");
+
+	/**
+	 * Creates the first book given to players in the war.
+	 * @return String containing the raw text of what will become a book.
+	 */
```
Comment (TheFlagCourier) [inline src/com/palmergames/bukkit/towny/war/eventwar/WarBooks.java:19]: Needs `@param war /description/`

**Review by TheFlagCourier — CHANGES_REQUESTED** [review]
Rebased off my local master and ran `mvn javadoc:javadoc`.
Here's the pain points I see.

**Pull request closed without merging**